mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Ink.Mixfile do
   def project do
     [
       app: :ink,
-      version: "0.8.1",
+      version: "0.8.2",
```

Comment:
You don't need to bump the version. All the dependencies you updated are marked as `only: :dev` or something else and therefore not included in the package anyway.

Reply:
Done